consensus/src/metrics.rs

```diff
@@ -61,3 +61,28 @@ impl Default for ConsensusMetrics {
         Self::new(default_registry())
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct ChannelMetrics {
+    /// occupancy of the channel from the `Consensus` to `SubscriberHandler`
```

Comment:
We might want to add a comment telling the reader to go look at the `tx_committed_certificates` and `tx_new_certificates` in the primary as well, as those are also relevant.